<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>674</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>420</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>420</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Font map</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::VLine</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="midLineWidth">
              <number>2</number>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="widgetResizable">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>396</width>
                <height>366</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnOpenFont">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Source Sans Pro</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveFont">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Source Sans Pro</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveAsFont">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Source Sans Pro</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Save as...</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Source Sans Pro</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Pixel view size:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="pixelSizeSpinBox">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Char/chars manipulation</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="5" column="0">
            <layout class="QVBoxLayout" name="pixelatorVLayout">
             <property name="topMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="lblCharInfo">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>9</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(146, 183, 158);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>None</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="pixelViewVLayout"/>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="horizontalSpacing">
              <number>4</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="6" column="1">
              <widget class="QToolButton" name="btnAddHeight">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::LeftArrow</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Left:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QToolButton" name="btnAddRight">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::LeftArrow</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Right:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QToolButton" name="btnAddLeft">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::LeftArrow</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QToolButton" name="btnRemoveLeft">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>-</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::RightArrow</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="3">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Shift Y:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Height:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QToolButton" name="btnRemoveHeight">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>-</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::RightArrow</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QToolButton" name="btnRemoveRight">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>-</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::RightArrow</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="3">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="5">
              <widget class="QToolButton" name="btnShiftRight">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::RightArrow</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="4">
              <widget class="QToolButton" name="btnShiftLeft">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::LeftArrow</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="5">
              <widget class="QToolButton" name="btnShiftUp">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::UpArrow</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="4">
              <widget class="QToolButton" name="btnShiftDown">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 85, 127, 96);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonIconOnly</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::DownArrow</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="3">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Shift X:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(146, 183, 158);
font: 10pt &quot;Open Sans&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>add/remove, shift:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QCheckBox" name="chBoxAllChars">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>aplly for all chars</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
